sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -68,6 +69,7 @@ private object CosmosConfigNames {
     "spark.cosmos.throughputControl.globalControl.renewIntervalInMS"
   val ThroughputControlGlobalControlExpireIntervalInMS =
     "spark.cosmos.throughputControl.globalControl.expireIntervalInMS"
+  val SynapseLinkedService = "spark.synapse.linkedService"
```

Comment:
OLTP spark connector on github is not supposed to know about this config name.

the synapse specific is leaking to OLTP source code. please remove.

Reply:
Resolved